src/content/docs/es/reference/errors/markdown-image-not-found.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Imagen no encontrada.
```

Comment:
I usually leave the title as is since most of the error searches are a copy and paste from the console.